src/renderer/app-bridge.ts

```diff
@@ -153,6 +153,9 @@ export class AppBridge {
                     ssInstance.handleMessageEvents(data);
                 }
                 break;
+            case apiCmds.setOrigin:
+                this.origin = data;
```

Comment:
This leads to a security risk. This will open a back door.

Reply:
Ok... so what could we do instead?

Reply:
We do not have any other way of doing this. Instead, we can just change it locally and test. We do not need to commit it to master.

Reply:
I think I have another solution now - I just pushed it so I guess it will show up here automatically?

Reply:
I think we should not change the `origin` what so ever. Or we should get this confirmed with the security team.

Reply:
But it is only changed if we run the demo, right?

Reply:
No, we can set the env when we run the application.
Like this -> 

`export ELECTRON_DEMO=true && /Applications/Symphony.app/Contents/MacOS/Symphony`

Now anyone can have access to all the content from other domains 😲